install/upgrade_to_2.13.php

```diff
@@ -48,6 +48,14 @@ public function upgrade(Migration $migration) {
       $this->addTargetValidationSetting();
       $this->addFormVisibility();
       $this->addRequestSourceSetting();
+      $this->addEntityOption();
+   }
+
+   public function addEntityOption() {
+      $table = 'glpi_plugin_formcreator_entityconfigs';
+
+      $this->migration->addField($table, 'is_search_issue_visible', 'integer', ['after' => 'is_header_visible', 'value' => '-2']);
+      $this->migration->addPostQuery("UPDATE `glpi_plugin_formcreator_entityconfigs` SET `is_search_issue_visible`= 1 WHERE `entities_id` = 0");
```

Comment:
these 2 lines must run if the field does not already exists. Currently, running the upgrade twice (after a failed attempt or a forced upgrade) may overwrite the setting for the root entity.